src/backend/core/management/commands/verify_tiered_storage.py

```
@@ -0,0 +1,528 @@
+            if not blob:
+                return True  # No blob to verify against
+
+            # Download and decrypt
+            with self.service.storage.open(obj_name, "rb") as f:
+                encrypted = f.read()
+
+            decrypted = self.service.decrypt(encrypted, blob.encryption_key_id)
+
+            # The decrypted content is still compressed
+            # We need to decompress to verify the original hash
+            import pyzstd
```

Ruff (0.14.14)

`import` should be at the top-level of a file

(PLC0415)